pkg/cloud/aws/aws.go

```diff
@@ -0,0 +1,33 @@
+package aws
+
+import (
+	"embed"
+
+	"github.com/openshift/cluster-cloud-controller-manager-operator/pkg/cloud/common"
+	appsv1 "k8s.io/api/apps/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func init() {
+	var err error
+	awsResources, err = common.ReadResources(awsFS, awsSources)
+	utilruntime.Must(err)
+}
+
+var awsResources []client.Object
+
+func GetResources() []client.Object {
+	resources := make([]client.Object, len(awsResources))
+	for i := range awsResources {
+		resources[i] = awsResources[i].DeepCopyObject().(client.Object)
+	}
+
+	return resources
+}
+
+//go:embed assets/*
+var awsFS embed.FS
+var awsSources = []common.ObjectSource{
+	{Object: &appsv1.Deployment{}, Path: "assets/deployment.yaml"},
+}
```

Comment:
i think it would be nice to group the global variables at the top of the file. imo, it makes it easier to find when looking at the source. not a blocker, just a suggestion

Reply:
also, i notice that the openstack variables are exported, should these be `AwsFS` and `AwsSources`?

Reply:
Rather the opposite, we only need to expose the `GetResources()` method.